### DISCUSSION TITLE
Add dynamic lighting

### DISCUSSION THREAD

**Discussion by haszidaj** [IrisShaders/Iris#434] [Ideas]
please add dynamic lighting like optifine so we can see in the dark without placing torch for example
its very useful with shaders on

**Comment by AlisterGontineac** [comment]
It seems to work for me, but I'm using Astralex shaders. Just wish I could change some settings in the shader

**Reply by WereWolfix** [reply]
> It seems to work for me, but I'm using Astralex shaders. Just wish I could change some settings in the shader

I have found a way to change the settings from the shaders files. Obviously you must have unzipped the zip file. I give an example with BSL_shader. Go inside the BSL / shaders / lib folder, inside you will find a file called setting.glsl, open it with a text editor program. All the settings preceded by // will be disabled by default so if you want to have them active you have to delete the // same thing vice versa if you want to disable them add the // in front of the setting

**Reply by AlisterGontineac** [reply]
Ah okay, so Iris can only load files from inside the zip.

**Comment by MimzyFang** [comment]
https://www.curseforge.com/minecraft/mc-mods/lambdynamiclights

**Reply by AlisterGontineac** [reply]
I already use lambdynamiclights. That doesn't change the shader settings though
